.zenodo.json

```diff
@@ -0,0 +1,41 @@
+{
+  "creators": [
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Wasser, Leah",
+      "orcid": "0000-000-000-000"
+    },
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Joseph, Max",
+      "orcid": "0000-000-000-000"
```

Comment:
0000-0002-7745-9990

Reply:
i can't seem to edit the file in-place so here is my ORCID: 0000-0003-2135-0168

Reply:
Thank you @mbjoseph  @joemcglinchy  both are now updated!!